ROADMAP.md

```diff
@@ -0,0 +1,156 @@
+OpenMDAO 2020 Development Roadmap
+=============================================
+Author: Justin S. Gray   
+Date: 10/24/2019  
+
+This document represents the OpenMDAO development team's perspective on what we feel are the most important areas to focus on for framework development in 2020. 
+It is intended to communicate our current perspective to the OpenMDAO user community, and to provide a basis for their feedback. 
+It is not intended to be all encompassing or a binding development plan, 
+but should be regarded as an very strong indication of the focus of the core development team. 
+
+There are four main topic areas that will be focused on: 
+- Releasing OpenMDAO V3.0
+- Model & Data Visualization 
+- New Differentiation Tools that Reduce User Effort
+- Establishing A Community Contribution Process
+
+----------------------------------------------
+----------------------------------------------
+----------------------------------------------
+# Releasing OpenMDAO V3.0
+
+In January 2020, we plan to release V3.0 of the framework. 
+This update will **NOT** represent significant change to the codebase or functionality, 
+but will include several small but important practical changes: 
+
+## 1) Droping support for Python 2.0, and supporting only python 3.6 and greater
+- Scipy and Numpy have already dropped Python 2 support for future releases. 
+- Will allow us to modestly simplify our codebase 
+
+## 2) Removing all current deprecations 
+- We've made a strong attempt not to break backwards compatibility through deprecations where possible, 
+but V3.0 will solidify the current primary (re: non deprecated) APIs as the only ones. 
+
+
+----------------------------------------------
+----------------------------------------------
+----------------------------------------------
+# Model & Data Visualization 
+
+Motivation: Provide tools that make it easier to quickly build and debug complex 
+models with 100's of components organized into 10's of groups in 10's of hierarchy layers
+
+Overall Goal: Rely heavily on data stored in the CaseRecorder for all visualization. 
+This makes the visualization more portable (i.e. can share case db with other) and 
+also makes it available after a model was run. 
+
+## 1) Metamodel viewer   
+
+### Goal:
+    - Allow users to inspect their MetaModel to check its accuracy/smoothness/etc 
+    - Encourage greater use of OM MetaModel components 
+
+### Potential Challenges:
+    - Current implementation based on Bokeh, and performance isn't as fast as we'd like yet 
+    - Some functionality requires live prediction from MetaModel instance, 
+      so we require a server process running in background. 
+      This means the functionality can't be built into the CaseRecorder
+
+### Notes: 
+    - initial capability was released in OpenMDAO V2.9.0
+
+
+## 2) Improved n2 model viz tool  
```

Comment:
`viz` -< `visualization`  (I think we want actual words here, rather than abbreviations/lingo?)

Also, since this is markdown, can/should we use ` n<sup>2</sup>` n<sup>2</sup> when describing it?